# This expresses temperatures in degrees Celsius.
int32 motor_temperature
int32 cpu_temperature